src/crwimage_int.cpp

```diff
@@ -1091,7 +1091,7 @@ namespace Exiv2 {
         case 0x0012: ifdId = canonPiId; break;
         }
         assert(ifdId != ifdIdNotSet);
-        DataBuf buf = packIfdId(image.exifData(), ifdId, pHead->byteOrder());
+        DataBuf&& buf = packIfdId(image.exifData(), ifdId, pHead->byteOrder());
```

Comment:
`packIfdId` is returning a normal object. Why are you using `&&` here?

Is it part of the automatic transformation or something that you added manually?

Reply:
manual. So the explicit change did more than the changes here. I reverted some of them. This was part of getting explicit working. I'll revert when I can.